knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -425,16 +426,21 @@ func (r *ReconcileKnativeKafka) buildManifest(instance *serverlessoperatorv1alph
 	var resources []unstructured.Unstructured
 
 	if build == manifestBuildAll || (build == manifestBuildEnabledOnly && instance.Spec.Channel.Enabled) || (build == manifestBuildDisabledOnly && !instance.Spec.Channel.Enabled) {
-		channelRBACProxy, err := monitoring.AddRBACProxySupportToManifest(instance, monitoring.KafkaChannelComponents)
+		rbacProxy, err := monitoring.AddRBACProxyToManifest(instance, monitoring.KafkaChannelController, monitoring.KafkaChannelWebhook)
 		if err != nil {
 			return nil, err
 		}
-		resources = append(resources, channelRBACProxy.Resources()...)
+		resources = append(resources, rbacProxy.Resources()...)
 		resources = append(resources, r.rawKafkaChannelManifest.Resources()...)
 	}
+
 	// Kafka Control Plane
 	if build == manifestBuildAll || (build == manifestBuildEnabledOnly && enableControlPlaneManifest(instance.Spec)) || (build == manifestBuildDisabledOnly && !enableControlPlaneManifest(instance.Spec)) {
-		// TODO: RBAC
+		rbacProxy, err := monitoring.AddRBACProxyToManifest(instance, monitoring.KafkaController, monitoring.KafkaWebhook)
+		if err != nil {
+			return nil, err
+		}
+		resources = append(resources, rbacProxy.Resources()...)
 		resources = append(resources, r.rawKafkaControllerManifest.Resources()...)
 	}
 
```

Comment:
Could you remove these leftovers: https://github.com/openshift-knative/serverless-operator/blob/ad20cccbe0f18fa8cdb0edc21aaf32be594f32c0/knative-operator/pkg/controller/knativekafka/knativekafka_controller.go#L449-L453.

Reply:
I will do this in the next PR since I need to test or tweak the existing dashboards for KafkaSource.